index.html

```diff
@@ -275,12 +275,12 @@
             for a subsequent requestPointerLock.
           </p>
 
-          <p>If any element (including this one) in the same document
+          <p>If any element (including this one) in the same shadow-including document
```

Comment:
"in a shadow-including document" is defined in DOM Standard, but strictly speaking, "in the same shadow-including document" is not defined, I think.

How about "If any element (including this one) who shares the same shadow-including root of the target"?